src/components/RubricEditor.vue

```
@@ -624,7 +667,16 @@ export default {
                     hasUnnamedCategories = true;
                 }
 
-                for (let j = 0, len2 = row.items.length - 1; j < len2; j += 1) {
+                const amountOfItems = row.items.length - (this.rowData[row.id].editable ? 1 : 0);
```

Maybe we should instead filter out all items that have no header, description, or points.

---

I don't agree with that, as that really changes how our the editor works. I do agree we should throw an error for every item that we will discard which has a header, description, id or points.